test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -35,7 +35,7 @@
     skipShapeChecking,
     skipTraceTest,
 )
-from torch import Tensor
+from torch import hub, Tensor
```

Comment:
```suggestion
from torch import Tensor
```

Since hub is use only once, I would just call `torch.hub._check_module_exists` where it is used to keep the code clean. `torch.hub` is not too long anyways.